frontend/src/screens/PersonProfile/PersonProfilePage.tsx

```diff
@@ -51,6 +58,27 @@ export default function PersonProfilePage() {
       );
     }
   };
+
+  const handleBan = () => {
+    if (isAuthorized) {
+      toggleUserBan(
+        otherUserId,
+        'bannedUser',
+        t('personalProfile.personalProfilePage.banningUpdated')
+      );
+    }
+  };
+
+  const handleUnban = () => {
+    if (isAuthorized) {
+      toggleUserBan(
+        otherUserId,
+        'user',
+        t('personalProfile.personalProfilePage.banningUpdated')
+      );
+    }
+  };
```

Comment:
it should toggle between:
- 'banned' and currentUserRole
not:
- 'banned' and 'user'

mb I'm wrong, explain please

Reply:
done